db/migrate/20200803104010_add_programme_status_to_activities.rb

```diff
@@ -0,0 +1,5 @@
+class AddProgrammeStatusToActivities < ActiveRecord::Migration[6.0]
+  def change
+    add_column :activities, :programme_status, :string
```

Comment:
Should existing activities be given a value for this field, and should it have a default value for new records? Or should the default be left as `NULL`?

Reply:
After reading the final commit I just realised this is one of those tricky things -- `programme_status` is ultimately required, but we save partially-complete activitieis during the multi-step form process, so we need to allow `NULL` in the database. We may still want to set a default value that reflects the concerns in the last commit here.

Reply:
I'm not sure what the best thing to do here is. Same as the conversation above about the `Replace me` placeholder. These both are things I haven't considered but that definitely need attention, and I will need help to make decisions. Probably better to have a conversation if we have time

Reply:
@MariluzHerrera I think we should leave it with the default set to `NULL` for now, but make a new card to remind us to fix this later (probably once `programme_status` has completely replaced `status`).

Any data manually-inputted through the front end will have this filled in correctly, so I don't think it's an issue.  Probably just need to check that an activity where `programme_status` is set to `Replace me` pre-populates the form with an empty selection when users attempt to edit it (so when they save they're force to choose something!)

Reply:
I'll create a new card for this